src/antares/service/local_services/link_local.py

```diff
@@ -133,3 +137,56 @@ def sort_link_properties_dict(ini_dict: Dict[str, str]) -> Dict[str, str]:
             "filter-year-by-year",
         ]
         return dict(sorted(ini_dict.items(), key=lambda item: dict_order.index(item[0])))
+
+    def get_capacity_direct(
+        self,
+        area_id: str,
+        second_area_id: str,
+    ) -> pd.DataFrame:
+        return read_timeseries(
+            TimeSeriesFileType.LINKS_CAPACITIES_DIRECT,
+            self.config.study_path,
+            area_id=area_id,
+            second_area_id=second_area_id,
+        )
+
+    def get_capacity_indirect(
+        self,
+        area_id: str,
+        second_area_id: str,
+    ) -> pd.DataFrame:
+        return read_timeseries(
+            TimeSeriesFileType.LINKS_CAPACITIES_INDIRECT,
+            self.config.study_path,
+            area_id=area_id,
+            second_area_id=second_area_id,
+        )
+
+    def get_parameters(
+        self,
+        area_id: str,
+        second_area_id: str,
+    ) -> pd.DataFrame:
+        return read_timeseries(
+            TimeSeriesFileType.LINKS_PARAMETERS, self.config.study_path, area_id=area_id, second_area_id=second_area_id
+        )
+
+    def read_links(self) -> list[Link]:
+        link_path = self.config.study_path / "input" / "links"
+        link_clusters = []
+        for element in link_path.iterdir():
+            area_from = element.name
+            for content_area in element.iterdir():
+                if (
+                    content_area.is_file()
+                    and content_area.suffix == ".txt"
+                    and content_area.stem.endswith("parameters")
+                ):
+                    link_clusters.append(
```

Comment:
You didn't do the part on LinkUi and LinkProperties. Not really a big deal if you add a TODO but you can do this inside this PR

Reply:
I did it with the aliases validation